packages/opentelemetry-instrumentation-google-generativeai/tests/test_generate_content.py

```
@@ -163,3 +165,9 @@ def assert_message_in_logs(log: LogData, event_name: str, expected_content: dict
     else:
         assert log.log_record.body
         assert dict(log.log_record.body) == expected_content
+
+def test_imports():
+    # Ensure heavy dependencies are not accidentally imported
+    import opentelemetry.instrumentation.google_generativeai
```

Ruff (0.12.2)

`opentelemetry.instrumentation.google_generativeai` imported but unused

Remove unused import: `opentelemetry.instrumentation.google_generativeai`

(F401)